AlDente/Helper.swift

```diff
@@ -248,7 +248,8 @@ final class Helper {
             print("helperVersion:", helperVersion, " version from helper:", version)
             if !helperVersion.elementsEqual(version) {
                 reply(false)
-                return()            }
+                return()
+            }
             else{
```

Comment:
The else statement could be fixed as well

```swift
if !helperVersion.elementsEqual(version) {
    reply(false)
    return()
} else {
```

See : https://google.github.io/swift/#braces

Reply:
Good point! Actually, I personally prefer the convention you suggested. 

However, upon reviewing the existing codebase of this project, it seems that the original author prefers the current way, which is why I didn't modify the else statement.